hadoop-ozone/client/src/test/java/org/apache/hadoop/ozone/client/TestHddsClientUtils.java

```diff
@@ -173,34 +222,6 @@ public void testBlockClientFallbackToScmNamesWithPort() {
     assertEquals(OZONE_SCM_BLOCK_CLIENT_PORT_DEFAULT, address.getPort());
   }
 
-  @Test
-  public void testClientFallbackToScmNamesNoPort() {
-    // When OZONE_SCM_CLIENT_ADDRESS_KEY is undefined, it should fallback
```

Comment:
I think, once we restore back to failback logic to ozone.scm.names, these tests will still be valid.

Reply:
Yes added them back